hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -4599,6 +4604,13 @@ func (r *HostedClusterReconciler) validateNetworks(hc *hyperv1.HostedCluster) er
 	return errs.ToAggregate()
 }
 
+func (r *HostedClusterReconciler) validateOCPConfigurations(ctx context.Context, hc *hyperv1.HostedCluster, client client.Client) error {
+	var errs field.ErrorList
+	errs = append(errs, validateOCPAPIServerConfiguration(ctx, hc, client)...)
```

Comment:
Why not just call `ValidateOCPAPIServerSANs` directly here? What's the point of the extra method?

Reply:
Maybe add a function definition to this function and include that as a TODO with a Jira ticket link?

Reply:
Created a ticket for that: [CNTRLPLANE-382](https://issues.redhat.com/browse/CNTRLPLANE-382)